web/src/pages/apd/new/HitechUpdateStatus.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ChoiceList } from '@cmsgov/design-system';
+import { Controller, useFormContext } from 'react-hook-form';
+
+const HitechUpdateStatus = ({ setUpdateStatus, updateStatus }) => {
```

Comment:
Hmm, have you tried using the `ApdUpdate.js` component? It's set up to handle both MMIS and HITECH types. I suspect it would need a slight modification to get the "type" passed in as a prop since it doesn't exist in the state yet, but would ideally be easier than creating two fully new components (`HitechUpdateStatus.js` and `MMISUpdateStatus.js`)

Reply:
I saw that but we want to get this merged so Tif had me create [4574](https://app.zenhub.com/workspaces/enterprise-cmcseapd-6390a3073de2e96379767873/issues/gh/enterprise-cmcs/eapd/4574) and so I included ApdUpdate :)